projects/1.16/assets/portality/portality/lang/zh_cn.json

```
@@ -1 +1,51 @@
-{}
\ No newline at end of file
+{
+    "itemGroup.portality": "Portality",
+    "block.portality.controller": "传送门控制器",
+    "block.portality.frame": "传送门框架",
+    "block.portality.module_interdimensional": "多维度交互模块",
+    "block.portality.module_items": "物品模块",
+    "block.portality.module_fluids": "流体模块",
+    "block.portality.module_energy": "能量模块",
+    "block.portality.generator": "低效能发电机",
+    "portality.controller.error.size": "错误：目前的传送门结构是无效的！",
+    "portality.controller.error.privacy": "错误：此传送门是私有的！",
+    "portility.controller.info.icon_changed": "传送门图标已更改！",
+    "portality.display.private": "私有",
+    "portality.display.public": "公共",
+    "portality.display.dial": "拨号链接",
+    "portality.display.dial_once": "拨号链接一次",
+    "portality.display.force": "强制链接",
+    "portality.display.call_portal": "调用传送门",
+    "portality.display.close_portal": "关闭传送门",
+    "portality.display.change_name": "更改传送门名称",
+    "portality.display.make_public": "将传送门转为公共",
+    "portality.display.make_private": "将传送门设为私有",
+    "portality.display.hide_name": "隐藏名称显示",
+    "portality.display.show_name": "显示名称显示",
+    "portality.display.change_color": "改变传送门颜色",
+    "portality.gui.controller": "名称：",
+    "portality.gui.controller.private": "私有：",
+    "portality.gui.controller.max_distance": "最大距离：",
+    "portality.gui.controller.interdimensional": "多维度交互：",
+    "portality.gui.controller.power": "能量：",
+    "portality.gui.controller.link": "链接：",
+    "portality.gui.controller.link_active": "活跃",
```

```suggestion
    "portality.gui.controller.link_active": "激活",
```
1.12.2如此译
原文：**Active**